tools/lang-tags-from-hb.py

```diff
@@ -82,6 +100,10 @@ def extract_languages(content):
 
             langdict[am_lang_ot] = {"html": am_lang_bcp, "name": am_lang_name}
 
+    for lr_lang in last_resort_langdict:
```

Comment:
Is it really necessary to keep track of the `last_resort_langdict` and only at the very end of processing replace items in the in the `langdict`? IMO the code would be easier to understand if it built up just once list (`langdict`). It seems that the logic for using a `last_resort` option is already present at line 75-78.

Reply:
They truly are last resort languages. I think this code is indeed somewhat clunky, but it does catch the logic of choosing a lannguage. Might revisit layer, thanks for spotting this though!